tests/test_onnxml_decision_tree_converter.py

```diff
@@ -251,19 +252,20 @@ def test_random_forest_multi_random(self):
         model.fit(X, y)
         self._test_classifier(X, model)
 
-    # Used for small tree tests
-    @unittest.skipIf(
-        not (onnx_ml_tools_installed() and onnx_runtime_installed()), reason="ONNXML test require ONNX, ORT and ONNXMLTOOLS"
-    )
-    def test_random_forest_classifier_single_node(self):
-        warnings.filterwarnings("ignore")
-        np.random.seed(0)
-        X = np.random.rand(1, 1)
-        X = np.array(X, dtype=np.float32)
-        y = np.random.randint(1, size=1)
-        model = RandomForestClassifier(n_estimators=5).fit(X, y)
-        model.fit(X, y)
-        self._test_classifier(X, model)
+    # # Used for small tree tests
+    # # Commenting this test for the moment because it hits a bug in ORT / ONNXMLTOOLS (https://github.com/onnx/onnxmltools/issues/415)
```

Comment:
Ah, this is why the coverage drop, right? thanks for filing the[ onnx issue](https://github.com/onnx/onnxmltools/issues/415)!

Reply:
This plus some corner condition I think.